addons/links/src/react/components/link.test.tsx

```diff
@@ -28,17 +28,21 @@ const mockChannel = () => {
     once: jest.fn(),
   };
 };
+const mockAddons = (addons as unknown) as jest.Mocked<typeof addons>;
 
 describe('LinkTo', () => {
   describe('render', () => {
     it('should render a link', async () => {
-      const channel = mockChannel();
-      addons.getChannel.mockReturnValue(channel);
+      const channel = mockChannel() as any;
+      mockAddons.getChannel.mockReturnValue(channel);
+
+      const { container } = render(<LinkTo kind="foo" story="bar" />);
 
-      const wrapper = shallow(<LinkTo kind="foo" story="bar" />);
-      await wrapper.instance().updateHref(wrapper.props());
-      wrapper.update();
-      expect(wrapper).toMatchSnapshot();
+      expect(container.firstChild).toMatchInlineSnapshot(`
+        <a
+          href="/"
```

Comment:
Shouldn't we have a `foo--bar ` somewhere because we are setting `kind="foo" story="bar"` on L39?

Reply:
Fixed, href is set asynchronously (componentDidMount)